lnrpc/rpc.proto

```diff
@@ -1704,6 +1704,9 @@ message OpenChannelRequest {
     carried out in an interactive manner (PSBT based).
     */
     FundingShim funding_shim = 14;
+
+    /// The maximum amount of coins in millisatoshi that can be pending within the channel.
+    int64 MaxValueInFlight = 15 [json_name = "max_value_in_flight_msat"];
```

Comment:
I haven't confirmed, but you may need to update your protobuf generation dependencies if you haven't already as we've updated them recently (see https://github.com/lightningnetwork/lnd/tree/master/lnrpc#generate-protobuf-definitions).